### PR TITLE
first draft of code review guidelines doc

### PR DESCRIPTION
**Pull request by diazrenata**
Initial draft of a document describing code review process.

### CODE REVIEW

**Review by Aariq — CHANGES_REQUESTED** [review]
This looks good but you need to remove the `code-review_files` directory and run `quarto render` to render the whole book instead of clicking render on the individual document.

File: code-review.qmd [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: "Code review"
+editor: source
+---
+
+## Code review goals
+
+Code review helps maintain software quality. It can also be a great way to share knowledge and create common standards and approaches across the group.
```
Comment (Aariq) [inline code-review.qmd:8]: one sentence per line makes for easier diffs.  If you merge main, this is set as default in _quarto.yml and just opening this doc, making a minor edit, and hitting "save" should do this for you.

Comment (diazrenata) [reply code-review.qmd]: Done!

**Review by Aariq — APPROVED** [review]

**Comment by diazrenata** [comment]
> This looks good but you need to remove the `code-review_files` directory and run `quarto render` to render the whole book instead of clicking render on the individual document.

Done!

**Comment by Aariq** [comment]
Do we want this in "Group Procedures" maybe?  I guess this could eventually be a service we offer, but for now I think it might fit better with the more "internal" stuff in "Group Procedures" instead of "Group Programming"

File: _quarto.yml [hunk 2]
```diff
@@ -32,6 +32,7 @@ book:
         - drop-in-hours.qmd
         - incubator.qmd
         - workshops.qmd
+        - code-review.qmd
```
Comment (Aariq) [inline _quarto.yml:35]: I think this fits better in the "Group Procedures" section

**Pull request merged**